From: Adrian Bunk <bunk@debian.org>
Date: Sun, 7 Jun 2020 15:41:19 +0200
Subject: Remove unused libipsec linking with libfl

Due to the yylex requirement this caused FTBFS
when a shared libfl is provided.
---
 src/libipsec/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libipsec/Makefile.am b/src/libipsec/Makefile.am
index 6a4e3b3..df1e106 100644
--- a/src/libipsec/Makefile.am
+++ b/src/libipsec/Makefile.am
@@ -26,7 +26,6 @@ libipsec_la_SOURCES = \
 # version is current:revision:age.
 # See: http://www.gnu.org/manual/libtool-1.4.2/html_chapter/libtool_6.html#SEC32
 libipsec_la_LDFLAGS = -version-info 0:1:0
-libipsec_la_LIBADD = $(LEXLIB)
 
 noinst_HEADERS = ipsec_strerror.h
 
